Neos.Flow/Classes/Http/Component/RequestBodyParsingComponent.php

```diff
@@ -32,6 +32,9 @@ class RequestBodyParsingComponent implements ComponentInterface
     public function handle(ComponentContext $componentContext)
     {
         $httpRequest = $componentContext->getHttpRequest();
+        if (!empty($httpRequest->getParsedBody())) {
```

Comment:
I have to check how we handled this before, but Guzzle ServerRequset::fromGlobals() sets $_POST as parsedBody, so if it was a "normal" POST request the parsed body will be fine already at this point.

Reply:
Before we just merged $_POST into arguments on Request creation and kept the request body around until we parsed it when mapping arguments for the ActionController. Guzzle is more correct in assigning the POST as parsed Body, if it exists. It's just a PHP inbuilt request body parsing for `multipart/form-data`/`application/x-www-form-urlencoded` after all.